src/dotnet/commands/dotnet-add/dotnet-add-package/AddPackageParser.cs

```diff
@@ -47,7 +47,9 @@ public static Command AddPackage()
                               LocalizableStrings.CmdPackageDirectoryDescription,
                               Accept.ExactlyOneArgument()
                                     .With(name: LocalizableStrings.CmdPackageDirectory)
-                                    .ForwardAsSingle(o => $"--package-directory {o.Arguments.Single()}")));
+                                    .ForwardAsSingle(o => $"--package-directory {o.Arguments.Single()}")),
+                Create.Option("--interactive",
```

Comment:
this map to a nuget.xplat interface right?

Reply:
i think there should be some code to pass it down.

Reply:
It should be something like this line https://github.com/nkolev92/cli/blob/c7cde12e2def4f962db8d72ee6a25e29b424f49f/src/dotnet/commands/dotnet-add/dotnet-add-package/Program.cs#L148

It is a pass through, so I don't think automatic tests are necessary, but please verify by run these command by hand

Reply:
This piece of code suggests I don't need to do it. 

https://github.com/nkolev92/cli/blob/c7cde12e2def4f962db8d72ee6a25e29b424f49f/src/dotnet/commands/dotnet-add/dotnet-add-package/Program.cs#L144-L146

I'll verify by manually building the cli + nuget.

Reply:
@wli3 
I added a forwarding call.